ui/src/component/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Pie from './pie';
+
+const chartMapper = {
+  pie: props => (<Pie {...props} />),
+};
```

Comment:
```
const chart2Component = {
   pie: 'Pie',
}

render() {
    const ChartComponent = _.result(chart2Component, chartType, 'line');
    return (<div>{  <ChartComponent {...props} }</div>);
}
```
Move chart initialize to render function, if we need to pass more parameters to chart,we only need modifying one place